pom.xml

```diff
@@ -67,6 +67,10 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
```

Comment:
In Spring Boot projects, it's usually better to opt for starters rather than individual dependencies (look for spring-boot-starter-validation)